README.md

```diff
@@ -32,7 +32,18 @@ To build for the EV3:
 
 * On your host computer (not in `brickstrap shell`), use NFS or sshfs to share
 <some-path-like-home/user/build-brickman> with your EV3.
-* On your EV3, connect the share and run `./brickman`
```

Comment:
brickman used to run itself on the next available tty, but does not do that any more, so this would not work anyway. My current method of debugging programs that need to use the LCD on the EV3 is with<https://gist.github.com/dlech/4c9b268571af59cb7bb4>. It essentially does the same thing that brickman does to launch programs, but redirects stdout and stderr to the remote tty from which it was launched.

So rather than messing with systemd stuff, I thing you can just run...

    G_MESSAGES_DEBUG=all sudo -- ~/runvt.sh ./brickman